extensions/amp-bind/0.1/bind-validator.js

```diff
@@ -33,9 +34,23 @@ let PropertyRulesDef;
  * @private {Object<string, ?PropertyRulesDef>}
  */
 const GLOBAL_PROPERTY_RULES = map({
+  'text': null,
```

Comment:
Yea, I originally had only whitelisted text elements for `[text]` but I think a more general solution is more consistent (`Node.textContent` is not limited to text elements) and makes the validator changes smaller.